src/web/components/CardHeadline.tsx

```diff
@@ -15,24 +15,24 @@ const fontStyles = (size: SmallHeadlineSize) => css`
 const underlinedStyles = (size: SmallHeadlineSize) => {
     function generateUnderlinedCss(baseSize: number) {
         return css`
-            display: inline;
             background-image: linear-gradient(
                 to bottom,
                 transparent,
                 transparent ${baseSize - 1}px,
                 rgba(199, 0, 0, 0.5)
             );
-            line-height: ${baseSize}px;
+            line-height: ${baseSize - 1}px;
```

Comment:
If the line-height is 1px smaller than the background, we’ll never see the last line’s underline. That also means there’s a 1px offset for every line.